chapter_convolutional-neural-networks/lenet_vn.md

```diff
@@ -72,7 +75,14 @@ Each convolutional layer uses a $5\times 5$ kernel and processes each output wit
 The first convolutional layer has 6 output channels, and second convolutional layer increases channel depth further to 16.
 -->
 
-*dịch đoạn phía trên*
+Các đơn vị cơ bản trong khối tích chập là một tầng tích chập và một lớp gộp trung bình tiếp theo
+(lưu ý rằng gộp lớn nhất hoạt động tốt hơn, nhưng nó chưa được phát minh vào những năm 90).
+Tầng tích chập được sử dụng để nhận dạng các mẫu không gian trong ảnh,
+chẳng hạn như các đường kẻ và các bộ phận của các đối tượng và lớp gộp trung bình tiếp theo được sử dụng để làm giảm tính chiều.
+Khối tầng tích chập gồm các ngăn xếp lặp lại của hai đơn vị cơ bản này.
+Mỗi tầng chập sử dụng bộ lọc có kích thước $5\times 5$ và xử lý mỗi đầu ra với một hàm kích hoạt sigmoid
+(một lần nữa, lưu ý rằng ReLUs hiện được biết là hoạt động đáng tin cậy hơn, nhưng đã chưa được phát minh).
```

Comment:
```suggestion
Mỗi tầng tích chập sử dụng bộ lọc có kích thước $5\times 5$ và xử lý mỗi đầu ra với một hàm kích hoạt sigmoid
(một lần nữa, lưu ý rằng ReLUs hoạt động đáng tin cậy hơn, nhưng chưa được phát minh tại thời điểm đó).
```

Reply:
hm.. mình chưa hiểu tại sao phải lược bỏ "_hiện được biết_" nhỉ? vì tác giả ghi là "_are now known to work more reliably_"
Ý mình là, việc lược bỏ có tác dụng gì nhỉ ngoài nghe "mượt" hơn vì mình thấy nó cũng bổ sung ý nghĩa về mặt tuyến tính hiện giờ-quá khứ. Chỉ là suy nghĩ của mình thôi nhé.

Reply:
Chỗ này mình thấy không có vấn đề gì nếu vẫn giữ `hiện được biết`.